Source/cmake/OptionsGTK.cmake

```diff
@@ -266,6 +266,7 @@ set(INTROSPECTION_INSTALL_TYPELIBDIR "${LIB_INSTALL_DIR}/girepository-1.0")
 SET_AND_EXPOSE_TO_BUILD(WTF_PLATFORM_QUARTZ ${ENABLE_QUARTZ_TARGET})
 SET_AND_EXPOSE_TO_BUILD(WTF_PLATFORM_X11 ${ENABLE_X11_TARGET})
 SET_AND_EXPOSE_TO_BUILD(WTF_PLATFORM_WAYLAND ${ENABLE_WAYLAND_TARGET})
+SET_AND_EXPOSE_TO_BUILD(WTF_PLATFORM_RPI ${USE_WPEWEBKIT_PLATFORM_RPI})
```

Comment:
Where is `USE_WPEWEBKIT_PLATFORM_RPI` defined?

Reply:
Currently it is on the buildroot config level. I know that you want to have rather on cmake level - so recognize building target during configuration and set USE_WPEWEBKIT_PLATFORM_RPI if it is RPI.